deploy/clowdapp.yaml

```diff
@@ -0,0 +1,92 @@
+# Copyright 2022 Red Hat, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: ccx-insights-results-aggregator-exporter
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: ccx-insights-results-aggregator-exporter
+    labels:
+      app: insights-results-aggregator-exporter
+  spec:
+    envName: ${ENV_NAME}
+    testing:
+      iqePlugin: ccx
+    jobs:
+    - name: insights-results-aggregator-exporter-job
+      parallelism: 1    
+      completions: 1    
+      activeDeadlineSeconds: ${ACTIVE_DEADLINE_SECONDS} 
+      backoffLimit: ${BACKOFF_LIMIT}
+      env:
+      # TODO: Set env
+      podSpec:
+        image: ${IMAGE}:${IMAGE_TAG}
+        # TODO: Use a command to export any database
+        command:
+          - ./irae
+          - "-show-authors"
+        restartPolicy: Never
+        resources:
+          requests:
+            cpu: ${CPU_REQUEST}
+            memory: ${MEMORY_REQUEST}
+          limits:
+            cpu: ${CPU_LIMIT}
+            memory: ${MEMORY_LIMIT}
+        env:
+        - name: ENV_NAME
+          value: ${ENV_NAME}
+    database:
+      name: ccx-data-pipeline
```

Comment:
you probably should use here this and drop name and version:
```
sharedDbAppName: ccx-insights-results
```
Then you need to specify dependencies and it should be:
```
dependencies:
  - ccx-insights-results
```

Reply:
what's `ccx-insights-results`?

Reply:
@juandspy that is clowdapp name for insights-results-aggregator.